src/main/java/uk/gov/companieshouse/overseasentitiesapi/model/dto/DueDiligenceDto.java

```diff
@@ -114,4 +116,16 @@ public String getDiligence() {
     public void setDiligence(String diligence) {
         this.diligence = diligence;
     }
+
+    public boolean isEmpty() {
```

Comment:
Not sure we need this isEmpty check - do we need to be this clever? Client should only submit one due diligence block. Checking that both due diligence blocks are present should be sufficient to fix the defect, as it's documented.

Would be good to get some other views on this though...

Reply:
See my original comment above, the web sends empty objects not null ones, so it would be difficult (or impossible) to implement this without that method.

Reply:
👍 missed that. Makes sense now.